controllers/grid/submissions/assignedSubmissions/AssignedSubmissionsListGridHandler.inc.php

```diff
@@ -63,17 +63,19 @@ protected function loadData($request, $filter) {
 		$authorDao = DAORegistry::getDAO('AuthorDAO');
 
 		list($search, $column, $stageId, $sectionId) = $this->getFilterValues($filter);
-		$title = $author = null;
+		$title = $author = $id = null;
 		if ($column == 'title') {
```

Comment:
Could you swap these out for a `switch`/`case` block? I find these a lot more readable.